contracts/NameWrapper.sol

```diff
@@ -280,6 +281,49 @@ contract NameWrapper is Ownable, ERC1155Fuse, INameWrapper {
         emit NameWrapped(ETH_NODE, label, wrappedOwner, _fuses);
     }
 
+    /**
+     * @dev Registers a new .eth second-level domain and wraps it.
+     *      Only callable by authorised controllers.
+     * @param label The label to register (Eg, 'foo' for 'foo.eth').
+     * @param wrappedOwner The owner of the wrapped name.
+     * @param duration The duration, in seconds, to register the name for.
+     * @param resolver The resolver address to set on the ENS registry (optional).
+     * @return expires The expiry date of the new name, in seconds since the Unix epoch.
+     */
+    function registerAndWrapETH2LD(
+        string calldata label,
+        address wrappedOwner,
+        uint duration,
+        address resolver,
+        uint96 _fuses
+    ) onlyController external override returns(uint expires) {
+        bytes32 labelhash = keccak256(bytes(label));
+        bytes32 node = _makeNode(ETH_NODE, labelhash);
+        uint256 tokenId = uint256(labelhash);
+
+        expires = registrar.register(tokenId, address(this), duration);
+
+        _checkFuses(ETH_NODE, _fuses);
```

Comment:
Not sure this exists on our new branch anymore?

And do we need to bother checking the fuses since it's just the ETH node? Can we just save the gas and assume trusted?

Reply:
It doesn't, but I needed to make the code work against master before I rebase it against our new PR once it's merged. :)